app/src/interfaces/input-rich-text-md/input-rich-text-md.vue

```diff
@@ -206,12 +206,12 @@ export default defineComponent({
 					value: props.value || '',
 				});
 
-				codemirror.on('change', (cm) => {
+				codemirror.on('change', (cm, { origin }) => {
+					if (origin === 'setValue') return;
+
 					const content = cm.getValue();
 
-					if (content !== props.value && (props.value === null && content === '') === false) {
-						emit('input', content);
-					}
+					emit('input', content);
```

Comment:
We'll have to double check what happens when `content == ''` and `props.value == null`. That cumbersome check was to make sure that the interface wouldn't stage an empty change on first load 🤔

Reply:
Shouldn't that only happen if codemirrors value is set manually by using 'setValue()'?

Reply:
Yeah! Just my PTSD from TinyMCE / codemirror kicking in